dist/ci/miniobs/Dockerfile

```diff
@@ -1,7 +1,7 @@
 #!BuildTag: osrt_miniobs
-FROM opensuse/leap:15.1
+FROM opensuse/leap:15.3
```

Comment:
this part is merged now - as 15.1 repo was removed

Reply:
good, thanks. So I think it is time to close this PR as having worker that do real build is still too long way which we in the end do not take.